[MS] AOSS-1424 setting feedback URL in the beta banner

szmg opened the pull request.

frankoid commented:
It would be nice to have a test for the unauthenticated URL - that could probably easily be added to ClientAcceptsAgentRequestFeature (which already visits a not-logged-in page). Otherwise looks good to me.

szmg commented:
You're right @frankoid, added the test and found out that it always points to the authenticated endpoint. Looked into other projects and could not find any occasions when this really reflects the current login status -- it shows unauthenticated one for pages that can be accessed without being logged in, otherwise the normal one. I've updated the PR.
